config.py

```diff
@@ -283,8 +283,77 @@ def logging_policy(cls):
     def localization_languages(cls):
         languages = cls.policy(cls.LOCALIZATION_LANGUAGES, default=["eng"])
         return [LanguageCodes.three_to_two[l] for l in languages]
+
+    # The last time the database configuration is known to have changed.
+    SITE_CONFIGURATION_LAST_UPDATE = "site_configuration_last_update"
```

Comment:
Why does this information need to be stored in the Configuration instance as well as the database?